### PR TITLE
Explore News Tiles & Explore Checkmarks

### PR DESCRIPTION
**Pull request by alexttyip**
- Added completed checkmarks to action tiles
- Cleanup
- Added checkmarks to campaign tiles
- Added news tiles

### CODE REVIEW

File: lib/models/news.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import 'package:app/models/Explorable.dart';
```
Comment (JElgar) [inline lib/models/news.dart:1]: Sorry, I should have mentioned, we currently have a class called "Article" that handles News stuff. That should match the api requirements.

**Review by JElgar — CHANGES_REQUESTED** [review]
This looks great, the only thing is there is already an article type so that can be used (and potentially renamed to News, what do you think is a better name?)

Other than that though this look fantastic thank you!

File: lib/models/news.dart (hunk 1 above)
Comment (alexttyip) [reply lib/models/news.dart:1]: I think Article is fine! Should I change everything else to Article (e.g. ExploreArticleTile) or leave it as is?

Comment (JElgar) [reply lib/models/news.dart]: Yes please!

**Review by JElgar — APPROVED** [review]
LGTM

**Pull request merged**